packages/optimize/README.md

```diff
@@ -0,0 +1,348 @@
+
+# React Native AEP Optimize Extension
+
+[![npm version](https://badge.fury.io/js/%40adobe%2Freact-native-aepoptimize.svg)](https://www.npmjs.com/package/@adobe/react-native-aepoptimize)
+[![npm downloads](https://img.shields.io/npm/dm/@adobe/react-native-aepoptimize)](https://www.npmjs.com/package/@adobe/react-native-aepoptimize)
+
+
+`@adobe/react-native-aepoptimize` is a wrapper around the iOS and Android [Adobe Experience Platform Optimize Extension](https://aep-sdks.gitbook.io/docs/) to allow for integration with React Native applications.
+
+## Peer Dependencies
+
+The Adobe Experience Platform Optimize extension has the following peer dependency, which must be installed prior to installing the optimize extension:
+- [Core](../core/README.md)
+- [Edge](../edge/README.md)
+
```

Comment:
Can you check the peer dependency for optimize, is messaging its peer dependency?  How about edgeIdentity?